ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -261,7 +273,12 @@ class AlertTabs extends Component {
           </div>
         </div>
 
-        <Tabs tabContentsClass="config-endpoint">
+        <Tabs
+          tabContentsClass="config-endpoint"
+          initialIndex={
+            locationState ? supportedConfigs[locationState.configName].index : 0
```

Comment:
as per above, `.index` is a bit strange. maybe using an Object method, or `.find`, or somehow the key name itself, would keep this lookup more contextually semantic for object?

Reply:
done as per above.